website/app/GeoKrety/Service/Config.php

```diff
@@ -198,4 +200,20 @@ public static function printEnvironements() {
         }
         echo \Markdown::instance()->convert($text);
     }
+
+    public function isValid() {
+        return count($this->validationDetails()) == 0;
+    }
+
+    public function validationDetails() {
+        $details = [];
+        if (GK_PASSWORD_HASH_ROTATION > GK_PASSWORD_MAX_HASH_ROTATION) {
+            array_push($details, 'GK_PASSWORD_HASH_ROTATION must be less than '.GK_PASSWORD_MAX_HASH_ROTATION);
```

Comment:
using syntax `sprintf('GK_PASSWORD_HASH_ROTATION must be less than %d', GK_PASSWORD_MAX_HASH_ROTATION)` would allow us to enable translation in a not intrusive way in the future.